extensions/stac-collection-spec.md

```diff
@@ -1,28 +1,29 @@
 # STAC Collection Extension Spec
 
-A group of STAC `Item` objects from a single source usually share a set of common metadata. This is especially true with satellite imagery that uses the STAC EO Extension. Rather than including these common metadata fields on every `Item`, they can be grouped into a `Collection`.
-
-### Warning
-This extension is still in its very first iteration, and likely will change as it gets use. Indeed there is a desire to combine it somehow with the static catalog
-'catalog.json' file, and to make sure it works very generically. There is a powerful and useful idea here, to be able to reference fields that don't need to repeat
-in every item. But the naming and fields may shift. Please try it out and give feedback on what works.
+A group of STAC `Item` objects from a single source can share a lot of common metadata. This is especially true with satellite imagery that uses the STAC EO Extension. Rather than including these common metadata fields on every `Item`, they can be grouped into a `Collection`.
 
 ## Collection Extension Description
 
 | element             | type info                 | name                    | description                                                                                 | 
 |----------------------|---------------------------|-------------------------|---------------------------------------------------------------------------------------------| 
-| collection_name | string | Collection Name | A name given to the Collection
-| collection_description | string (optional) | Collection Description | A human readable description of the collection
+| collection | string | Collection Name | A name given to the Collection
+| description | string (optional) | Collection Description | A human readable description of the collection
+
+A `Collection` does not have many specific fields, as it may contain any fields that are in the core spec as well as any other extension. This provides maximum flexibility to data providers, as some the set of common metadata fields can vary between different types of data. For instance, Landsat and Sentinel data always has a eo:off_nadir value of 0, because those satellites are always pointed downward (i.e., nadir), while satellite that can be pointed will have varying eo:off_nadir values.
 
```

Comment:
Landsat-8 may acquire off-nadir images for special collections, see:
https://landsat.usgs.gov/landsat-8-l8-data-users-handbook-section-2
"but special collections may be scheduled off-nadir." 
It seems to me that eo:plaform or eo:collections are better to be used as examples of fields that do not change.